src/components/wallet/NewMnemonicConfirm.tsx

```diff
@@ -0,0 +1,214 @@
+import React, { useState } from 'react'
+import { Card, Col, Row, Button, Input, Form } from 'antd'
+import { RedoOutlined } from '@ant-design/icons'
+import { Store } from 'antd/lib/form/interface'
+
+type WordType = {
+  text: string
+  _id: string
+  sequence?: number
+  error?: boolean
+  selected?: boolean
+}
+const mnemonic = 'real debris regret sea auto random agree police uncover gloom cloud ribbon'
+
+const MnemonicConfirmScreen: React.FC = (): JSX.Element => {
+  const [wordsList, setWordsList] = useState<WordType[]>([])
+  const [shuffledWordsList, setShuffledWordsList] = useState<WordType[]>([])
+  const [loadingMsg] = useState<string>('')
+  const [mnemonicError, setMnemonicError] = useState<string>('')
+  const [initialized, setInitialized] = useState<boolean>(false)
+
+  function init() {
+    const words = mnemonic.split(' ')
+    if (words && !initialized) {
+      const res = words.map((e: string) => {
+        const uniqueId = Math.random().toString(36).substring(2) + Date.now().toString(36)
+        return { text: e, _id: uniqueId }
+      })
+      setWordsList(res)
+      setShuffledWordsList(shuffledWords(res))
+      setInitialized(true)
+    }
+  }
+
+  function shuffledWords(words: WordType[]) {
+    const shuffler = (arr: WordType[]) => {
+      const newArr = arr.slice()
+      for (let i = newArr.length - 1; i > 0; i--) {
+        const rand = Math.floor(Math.random() * (i + 1))
+        ;[newArr[i], newArr[rand]] = [newArr[rand], newArr[i]]
+      }
+      return newArr
+    }
+    return words.length > 0 ? shuffler(words) : []
+  }
+
+  function sortedSelectedWords(): WordType[] {
+    function compare(a: WordType, b: WordType) {
+      const num1 = a.sequence
+      const num2 = b.sequence
+      let comparison = 0
+      if (num1 && num2 && num1 > num2) {
+        comparison = 1
+      } else if (num1 && num2 && num1 < num2) {
+        comparison = -1
+      }
+      return comparison
+    }
+    return wordsList.filter((e: WordType) => e.selected === true).sort(compare)
+  }
+
+  init()
+
+  const isSelected = (id: string) => {
```

Comment:
Please extract `isSelected` from component to export it and to add tests for it. It needs to be `isSelected = (id: string, wordsList)` then. Can be done w/ another PR.